src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -53,6 +53,23 @@ public void MakeCompilationOutputRunnable()
             ExportRuntimeAssets();
         }
 
+        private void VerifyCoreclrPresenceInPackageGraph()
```

Comment:
nit: the capitalization of Coreclr doesn't match the way you have it below in LibCoreClrBinaryNames.